config/locales/views/privacy/ko.yml

```
@@ -0,0 +1 @@
+ko:
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)